libdnf/rpm/solv/map_iterator.hpp

```diff
@@ -50,7 +50,7 @@ class SolvMapIterator {
     PackageId operator*() const { return current_value; }
 
     SolvMapIterator & operator++();
-    SolvMapIterator operator++(int) { return ++(*this); }
+    SolvMapIterator operator++(int);
```

Comment:
I am sorry, this operator is `iterator++`, or `++iterator`?

Reply:
`SolvMapIterator & operator++();` overloads `++iterator`.
`SolvMapIterator operator++(int);` overloads `iterator++`.